Cargo.toml

```diff
@@ -2,9 +2,9 @@
 name = "deltachat"
 version = "1.76.0"
 authors = ["Delta Chat Developers (ML) <delta@codespeak.net>"]
-edition = "2018"
+edition = "2021"
 license = "MPL-2.0"
-resolver = "2"
+rust-version = "1.56"
```

Comment:
What does this line do?

Reply:
It is MSRV, newer versions of cargo supports it and will exit with error if the version of rust is too small: https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field

This is not really useful because the field is only supported since 1.56, but later it will become useful by telling you explicitly when your rust version is too old instead of failing with an error during compilation because of some unsupported feature.